components/table/index.en-US.md

```diff
@@ -194,7 +194,7 @@ One of the Table `columns` prop for describing the table's columns, Column has t
 | sortDirections | Supported sort way, override `sortDirections` in `Table`, could be `ascend`, `descend` | Array | \[`ascend`, `descend`] |  |
 | sorter | Sort function for local sort, see [Array.sort](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort)'s compareFunction. If you need sort buttons only, set to `true` | function \| boolean | - |  |
 | sortOrder | Order of sorted values: `ascend` `descend` `null` | `ascend` \| `descend` \| null | - |  |
-| sortIcon | Customized sort icon | (props: { sorterOrder }) => ReactNode | - |  |
+| sortIcon | Customized sort icon | (props: { sorterOrder }) => ReactNode | - | 5.6.0 |
```

Comment:
感觉这里的 `sorterOrder ` 究竟是什么类型的是不是可以注明一下

Reply:
上面有：

<img width="842" alt="image" src="https://github.com/ant-design/ant-design/assets/49217418/02ef969a-8cbe-4f11-bfe0-8d8b615a67fa">

Reply:
主要是两个字段名其实是不一样的，不好对应上，看起来有点别扭

Reply:
emmm，我感觉没啥必要，已经注明 `sorterOrder` 类型了，这个具体类型在其它地方也有注明，文档中这种场景挺多的，如果都要注明的话，那要改的地方太多了

Reply:
@kiner-tang 

I agree with you. 
We should refactor the name of it on docs and useSorter.tsx. 

In this PR https://github.com/ant-design/ant-design/pull/42498#issuecomment-1556604000, I didn't do that, because I've thought it was better to fix in another PR.